graphql/server.go

```diff
@@ -226,11 +248,32 @@ func (c *conn) handleMutate(id string, mutate *mutateMessage) error {
 
 		start := time.Now()
 		c.logger.StartExecution(ctx, tags, true)
-		current, err := e.Execute(ctx, c.schema.Mutation, c.schema.Mutation, query)
+
+		output := &ComputationOutput{
+			Metadata: make(map[string]interface{}),
+		}
+
+		var middlewares []MiddlewareFunc
```

Comment:
While you're here, could you maybe try and de-duplicate these two functions? They're starting to bother me a lot.